manifest.json

```diff
@@ -368,5 +368,5 @@
   "name": "bids-fmriprep",
   "source": "https://github.com/nipreps/fmriprep",
   "url": "https://github.com/flywheel-apps/bids-fmriprep/blob/master/README.md",
-  "version": "1.1.22_20.2.4"
+  "version": "1.2.22_20.2.4"
```

Comment:
Should that be 1.2.0_20.2.4?  Changing the minor version should start the patch version number over at zero?